andela/archangel-backend
# #167730617 Sequelize setup and model file

#### What does this PR do?
Have sequelize configured in the Barefoot Nomad Application
#### Description of Task to be completed?
- Have to use sequlize as our ORM on the API. 
- Set up the migrations and models structure,
#### How should this be manually tested?
Clone the repo, then check for the file called `package.json` to find `sequelize` in the `dependencies`. After, go to the root folder and open a file `.sequelizerc` to find the configuration that created the `boiler plates` of models in the `database` folder.
#### Any background context you want to provide?
- create sequelizerc file to auto generate boiler plates codes
- edit the generated code to work with dotenv config output
#### What are the relevant pivotal tracker stories?
[#167730617](https://www.pivotaltracker.com/story/show/167730617)
